docs/api-reference.md

```diff
@@ -242,6 +223,9 @@ Example:
 
 ## Deprecated API
 
+### ConnectionNormalizer and ConnectionMatcher JavaSscript files
+The JavaScript files for connection normalizer and connection matcher are deprecated as of Tableau 2020.3. This means the element  `<script file="fileName.js"/>` (which was added inside the `<connection-matcher>` and `<connection-normalizer>` element) and the `<connection-matcher>` element itself are deprecated as of 2020.3. The `<connection-normalizer>` component can be added to the connectionResolver.tdr fie as shown in the connection-normalizer section above.
```

Comment:
```suggestion
### ConnectionNormalizer and ConnectionMatcher JavaSscript files
The JavaScript files for connection normalizer and connection matcher are deprecated as of Tableau 2020.4. This means the element  `<script file="fileName.js"/>` (which was added inside the `<connection-matcher>` and `<connection-normalizer>` element) and the `<connection-matcher>` element itself are deprecated as of 2020.4. The `<connection-normalizer>` component can be added to the connectionResolver.tdr fie as shown in the connection-normalizer section above.
```

This says the normalizer and matcher are deprecated as of 2020.3, shouldn't this be 2020.4?

Reply:
2020.3 is correct.  Coincides with bulk of the Gallery work and those policies.